### PR TITLE
Add crashlytics

### PR DESCRIPTION
**Pull request by androbbi**
I added Firebases's Crashlytics feature. It allows for crashes to be logged onto the Firebase console to facilitate development.

I have a test button in MainActivity that forces a crash that the Firebase Console will tell you about. It's purpose is to demonstrate the feature, feel free to remove that bit of code if you do not need it.

### CODE REVIEW

**Comment by androbbi** [comment]
My first commit was written before I saw the rules set for committing. I went to change it into the second commit title, but the first commit title ended up staying anyways. Sorry about that.

File: android/app/local.properties [hunk 1]
```diff
@@ -1,11 +1,8 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file must *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
```
Comment (rustema) [inline android/app/local.properties]: Can you remove this file from the pull request please?

File: android/app/src/main/java/io/peggyjo/peggymeter/FirstTimeActivity.java [hunk 2]
```diff
@@ -30,7 +33,21 @@ protected void onCreate(Bundle savedInstanceState) {
             controller.getSettingAdapter().setProperty(OPT_IN_PROPERTY, false);
             finish();
         });
+
         ((WebView) findViewById(R.id.webView)).loadUrl("file:///android_asset/welcome.html");
+
+        // Adds a button that will crash the app
```
Comment (rustema) [inline android/app/src/main/java/io/peggyjo/peggymeter/FirstTimeActivity.java]: Can you put this code in
  if( BuildConfig.DEBUG ){
  }

so it runs in debug mode only?

File: android/app/src/main/java/io/peggyjo/peggymeter/FirstTimeActivity.java [hunk 3]
```diff
@@ -36,18 +36,22 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        crashButton.setOnClickListener(new View.OnClickListener() {
-            public void onClick(View view) {
-                Crashlytics.getInstance().crash(); // Force a crash
-            }
-        });
-        addContentView(crashButton,
-                new ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
-                        ViewGroup.LayoutParams.WRAP_CONTENT));
+        if(BuildConfig.DEBUG)
+        {
```
Comment (rustema) [inline android/app/src/main/java/io/peggyjo/peggymeter/FirstTimeActivity.java]: Can you move { to the previous line to match style please?

**Pull request merged**